api/nodes/serializers.py

```diff
@@ -75,6 +77,9 @@ class NodeSerializer(JSONAPISerializer):
     files = JSONAPIHyperlinkedIdentityField(view_name='nodes:node-providers', lookup_field='pk', lookup_url_kwarg='node_id',
                                              link_type='related')
 
+    comments = JSONAPIHyperlinkedIdentityField(view_name='nodes:node-comments', lookup_field='pk', lookup_url_kwarg='node_id',
+                                               link_type='related', meta={'unread_comments_count': 'get_unread_comments_count'})
```

Comment:
Oh, I see you've made it so that the unread count is opt-in. I think this is a good idea, but ideally we wouldn't have to hardcode the key names here: `https://github.com/samanehsan/osf.io/blob/634e73eace3749faab9799a428a603fb5bc6912d/api/base/serializers.py#L161`. 

If that is most practical solution for now, perhaps add a TODO on the line above to refactor.